rust-toolchain.toml

```
@@ -0,0 +1,4 @@
+[toolchain]
+channel = "1.70.0"
```

1.70.0 is affected by [CVE-2023-38497](https://groups.google.com/g/rustlang-security-announcements/c/PEZQleQ6IUY).

```suggestion
channel = "1.71.1"
```